eslint.config.mjs

```diff
@@ -28,7 +46,22 @@ export default tseslint.config(
     },
     languageOptions: {
       parserOptions: {
-        projectService: true,
+        projectService: {
+          // We must enumerate each file that we want to lint that is not explicitly captured by a project-specific tsconfig
+          // allowDefaultProject does not allow `**` globs.
+          // https://github.com/typescript-eslint/typescript-eslint/issues/9739
+          allowDefaultProject: [
+            ...allProjectDirs.map(dir => `${dir}/*.ts`),
```

Comment:
You don't need this entry?
...allProjectDirs.map(dir => `${dir}/*.tsx`)

Reply:
I don't think we have any top-level `tsx` files. So we shouldn't need it for now